neuraxle/steps/encoding.py

```diff
@@ -0,0 +1,63 @@
+"""
+Encoding Pipeline Steps
+====================================
+You can find here encoder pipeline steps, for example, one hot encoder.
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+import numpy as np
+
+from neuraxle.base import NonFittableMixin, BaseStep
+
+
+class OneHotEncoder(NonFittableMixin, BaseStep):
```

Comment:
I am tempted to put this in the Numpy file instead. It feels alone here to be just this in one file. What do you think? or would there be a way to add that to the numpy steps file and then to split that file in many other files?

Reply:
moved to numpy.py